pom.xml

```diff
@@ -129,6 +154,11 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>3.0.0-M5</version>
+            </plugin>
```

Comment:
Why do we need this now but we didn’t before?

Reply:
Added in comment, but here is there reason:

Without the plugin, it is possible that tests won't run, if Maven version is not the latest on target platform.

so when I run `mvn test` I get below output since maven version is not latest:
```
Results :

Tests run: 0, Failures: 0, Errors: 0, Skipped: 0
```